src/docker-cache/config/docker-cache.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+enabled: false
+storage_backend: "azure"
+azure_account_name: ""
+azure_account_key: ""
+azure_container_name: dockerregistry
+fs_mount_path: "/var/lib/registry"
+registry_listener: ":5000"
+container_port: 5000
+service_port: 5000
+service_nodeport: 30500
+remote_url: https://registry-1.docker.io
+registry-htpasswd: dGVzdDokMnkkMDUkRUZiaWphaHovMHl4UC5xMFk1VW52TzljU2hHMThCRzM3QzBHNFhoRmFtTXdTUXdQLjBqQi4KCg==
```

Comment:
Is it a default passwd? What is the exact meaning of this string?

Reply:
Please check this.

Reply:
It's test test processed by htpasswd tool and base64 encoded, commented out in new commit.